install/Makefile.am

```diff
@@ -19,6 +19,10 @@ SUBDIRS =			\
 	$(NULL)
 
 install-exec-local:
+	mkdir -p $(DESTDIR)$(IPA_SYSCONF_DIR)/custodia
+	chmod 700 $(DESTDIR)$(IPA_SYSCONF_DIR)/custodia
```

Comment:
I wanted to be consistent but good point, I will change it to $(INSTALL). `-o root ` cannot be used as explained below. Changing file ownership is job of package manager.